test/integration/hotrestart_test.sh

```diff
@@ -4,13 +4,15 @@ set -e
 
 [[ -z "${ENVOY_BIN}" ]] && ENVOY_BIN="${TEST_RUNDIR}"/source/exe/envoy-static
 
-# TODO(htuch): Clean this up when Bazelifying the hot restart test below. At the same time, restore
-# some test behavior lost in #650, when we switched to 0 port binding - the hot restart tests no
-# longer check socket passing. See #654.
+# TODO(htuch): Clean this up when Bazelifying the hot restart test below.In this test script,
```

Comment:
Nit: We have already Bazelifyed the hot restart test, so you can drop the first sentence here.